toolbox/fdc3-compliance/src/test/fdc3.addIntentListener.ts

```diff
@@ -0,0 +1,10 @@
+describe("fdc3.addIntentListener", async () => {
+  it("Method is callable", async () => {
+    await window.fdc3.addIntentListener(
```

Comment:
Shouldn't we also add clean up logic (e.g. inside of an `afterEach()`) to prevent memory leaks?

Reply:
We will leave the await in place, given that it isn't causing problems and it will be async in the near future. 
We will also implement the suggested clean-up logic for listeners.